examples/pipelines/datacomp/components/load_from_hf_hub/fondant_component.yaml

```diff
@@ -32,6 +32,13 @@ produces:
       clipl14score:
         type: float32
 
+  textembedding:
```

Comment:
Wondering whether it won't be more intuitive to have the embeddings as a field rather than a subset. It makes more sense since different modalities (subsets) can have different embeddings, Wdyt?

Reply:
Yes I know, however image data and image embeddings are both heavy in terms of data, which is probably why the `subsets` were created in the first place. So if both image data and embeddings are part of the same subset, then Fondant will first always load both first, and then only provide the embeddings if the user specified this, right?

cc @RobbeSneyders